SD2019StackOverflowApp/src/main/java/ro/utcn/sd/mca/SD2019StackOverflowApp/exception/InvalidSOUserIdException.java

```diff
@@ -0,0 +1,8 @@
+package ro.utcn.sd.mca.SD2019StackOverflowApp.exception;
+
+public class InvalidSOUserIdException extends Exception {
+    @Override
+    public String getMessage() {
+        return "Invalid sOUserId!";
```

Comment:
Also, instead of overriding the getMessage method, you can simply create a  no-arguments constructor and pass the message to the super constructor.